src/components/SequenceSlide.vue

```diff
@@ -0,0 +1,116 @@
+<template>
+  <div>
+    <div class="slide-inner slide-light">
+      <div class="copy show-desktop">
+        <p>{{ slide.copyA }}</p>
+        <p>{{ slide.copyB }}</p>
+      </div>
+      <div class="copy show-mobile">
+        <p>{{ slide.copyA }} {{ slide.copyB }}</p>
+      </div>
+      <div class="image">
+        <img :src="this.slideImageSrc" />
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
+  name: "SlickSlideTemplate",
+  props: {
+    slide: Object,
+    name: String,
+  },
+  computed: {
```

Comment:
Computed() over data() is this because there are no changes being made to the data so we only wanted to have it loaded in once?

Reply:
We only want it to change IF the dependancies change